### PR TITLE
adds readme file and issue template for review

### PR DESCRIPTION
**Pull request by Philo**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+* **I'm submitting ...**
```
Comment (lynchjames) [inline .github/issue_template.md]: Can we also include questions on this list?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+
+The platform offers a set of developer components, documentation and features that give others the freedom to innovate at speed, at the service level.
+
+The Lenus Health platform is constantly evolving, new features and improvements are always in the pipeline and it is our goal to get those features into the hands of developers and end-users at semi-regular intervals throughout the year.
+
+Whilst we have a very long list of features and improvements we are planning to implement it is vitally important that we provide a forum for developers and end-users to discuss features that either we are planning or that you would like us to consider for implementation...this is that forum.
+
+We would love for those using the platform to submit any feedback about any aspect of the platform to us so we can:
+
+- better understand the features of the platform that you like (or dislike)
```
Comment (lynchjames) [inline README.md]: Can we mention asking questions here as well?

File: .github/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
```
Comment (lynchjames) [inline .github/issue_template.md:2]: Can we also include a question template?

**Review by lynchjames — CHANGES_REQUESTED** [review]
Pulling out questions specifically a little more.

**Comment by Philo** [comment]
Updates readme and template with requested updates @lynchjames

**Pull request merged**